### ISSUE TITLE
Recent releases missing on crates.io

### ISSUE THREAD

**Issue by alexanderkjall** [tuna-f1sh/cyme#40]
The last release there was 1.7.0: https://crates.io/crates/cyme

**Comment by tuna-f1sh** [comment]
Yes this is something I’m consciously holding back and noted in the [README](https://github.com/tuna-f1sh/cyme/tree/main?tab=readme-ov-file#crate) and #24. There have been some big API changes and wanted to wait for a major release with #26. It’s not ideal but even once that is merged, I still can’t publish until some dependancies are updated as it currently relies on some patches - which prevent `cargo publish`.

It’s not something I’m that happy with but can’t see a solution at the moment. I see it creates issues if one uses `cargo install cyme` - I’ll update the README to suggest `cargo install cyme —git https://github.com/tuna-f1sh/cyme` or something for now.
